src/ui/panels/quests.js

```diff
@@ -0,0 +1,232 @@
+import { CONSTANTS } from "../../constants/constants.js";
```

Comment:
FYI, if you rename the file then do no (or only minor changes) then git add the "deleted" file it will register as a name change rather than deletion + creation.

Reply:
🤔

Reply:
As a separate commit I mean